### ISSUE TITLE
Can't disable cache

### ISSUE THREAD

**Issue by eduardomozart** [lpaulsen93/dokuwiki-plugin-odt#303]
The plugin seems to use a similar logic to DW2PDF to check cache availability, as can be seen in: https://github.com/lpaulsen93/dokuwiki-plugin-odt/blob/7fe18af1f74963aa355252487538d8e9671863f7/action/export.php#L200

But there's no button to enable/disable ODT cache functionality and even implementing it I wasn't able to disable it completely (it always download from cache instead of generating a new ODT file each try).

**Comment by Klap-in** [comment]
The dw2pdf plugin has the config setting `usecache` to let in the Config Manager select the preference.
This can be copied from that plugin, including translated strings.

**Comment by Klap-in** [comment]
Indeed, the odt plugin copied a big part of the export function from dw2pdf plugin.

**Comment by Klap-in** [comment]
`~nocache~` is for DokuWiki to instruct the renderer. So it will not change the caching of the export. I maintain the dw2pdf plugin, so I seen quite a bit of that code. Do you have somewhere a branch with this code? Then I can have a look.

**Comment by eduardomozart** [comment]
I'll recreate my draft adding the ``odt>usecache`` directive to ODT plug-in again and send you a PR for us to debug it. I compared it with DW2PDF cache control mechanism that uses a logic similar to ``usecache`` from ODT plug-in but I was unable to figure out why DW2PDF cache is disabled when ``dw2pdf>usecache`` is disabled on plug-in settings but when disabling ``odt>usecache`` it's still downloading ODT file from cache.
